lib/mayaUsd/ufe/UsdUndoMaterialCommands.h

```diff
@@ -105,6 +111,11 @@ class MAYAUSD_CORE_PUBLIC UsdUndoAssignNewMaterialCommand : public Ufe::InsertCh
     //! sdrShaderIdentifier and assigns it to \p parentItem
     static UsdUndoAssignNewMaterialCommand::Ptr
     create(const UsdSceneItem::Ptr& parentItem, const std::string& sdrShaderIdentifier);
+    //! Create a UsdUndoAssignNewMaterialCommand that creates a new material based on \p
```

Comment:
Not clear about Doxygen but I would have move `\p` close to the parameter name `sdrShaderIdentifier`